packages/page-accounts/src/Accounts/index.spec.ts

```diff
@@ -332,33 +313,65 @@ describe('Accounts page', () => {
     return balanceOf(amountInt.toString() + decimalsPadded);
   };
 
-  const defaultFreeBalance = 1000;
-
-  const anAccount = (address: string = aliceAddress): AccountOverrides => ({
-    address,
-    balance: {
-      freeBalance: balance(defaultFreeBalance)
-    }
-  });
+  const anAccount = (): AccountOverrides => ({});
 
-  const anAccountWithBalance = (address: string, balance: { [P in keyof DeriveBalancesAll]?: DeriveBalancesAll[P] }) => ({
-    address,
+  const anAccountWithBalance = (balance: { [P in keyof DeriveBalancesAll]?: DeriveBalancesAll[P] }) => ({
     balance
   });
 
-  const anAccountWithInfo = (address: string, info: { [P in keyof UseAccountInfo]?: UseAccountInfo[P] }) => ({
-    address,
-    balance: {
-      freeBalance: balance(defaultFreeBalance)
-    },
+  const anAccountWithInfo = (info: { [P in keyof UseAccountInfo]?: UseAccountInfo[P] }) => ({
     info
   });
 
+  const anAccountWithStaking = (staking: { [P in keyof DeriveStakingAccount]?: DeriveStakingAccount[P]}) => ({
+    staking
+  });
+
   function assertColumnNotExistInTable (columnName: string, table: HTMLElement) {
     expect(within(table).queryByRole('columnheader', { name: columnName })).toBeFalsy();
   }
 
   function assertColumnExistsInTable (columnName: string, table: HTMLElement) {
     expect(within(table).getByRole('columnheader', { name: columnName })).toBeTruthy();
   }
+
+  function showBalance (amount: number): string {
+    return format(balance(amount));
+  }
+
+  const defaultAddresses = [
+    aliceAddress,
+    bobAddress,
+    charlieAddress
+  ];
+
+  function renderAccountsWithDefaultAddresses (...overrides: AccountOverrides[]): void {
```

Comment:
The three `render...` methods here look pretty much alike. Maybe one could be "base" and other would call it with different params? Or perhaps the `for` loops could be replaced with `.map` on the arrays.

Reply:
✅